libquiche.a does not contain symbols in quiche.h

I'm trying to build an libquiche.a for android, and followed the step in README, and it built successfully:
```
QITMAC000909deMac-mini quiche % cargo ndk -t armv7-linux-androideabi -p 21 -- build
[2021-02-05T13:09:42Z INFO  cargo_ndk::cli] Using NDK at path: /Users/liuyafei/Library/Android/sdk/ndk/21.1.6352462
[2021-02-05T13:09:42Z INFO  cargo_ndk::cli] NDK API level: 21
[2021-02-05T13:09:42Z INFO  cargo_ndk::cli] Building targets: armeabi-v7a
[2021-02-05T13:09:42Z INFO  cargo_ndk::cli] Building armeabi-v7a (armv7-linux-androideabi)
    Finished dev [unoptimized + debuginfo] target(s) in 0.19s
```

then I took `libquiche.a` in `quiche/target/armv7-linux-androideabi/debug` into my ndk project, but it said `rror: undefined reference to 'quiche_enable_debug_logging(void (*)(char const*, void*), void*)'`, I use `nm` to check the symbols in the lib, and it turns out no symbols are related to "logging":

```
QITMAC000909deMac-mini debug % nm libquiche.a |grep logging
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
no symbols
```

so I guess it's not the C/C++ mangling problem, but somehow that there are some symbols missing in libquiche.a which are defined in quiche.h

what did I do wrong?

--- a/.github/workflows/stable.yml
+++ b/.github/workflows/stable.yml
@@ -421,6 +421,6 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: ndk
-          args: -t ${{ matrix.arch }} -p ${{ env.NDK_LTS_VER }} -- build --verbose
+          args: -t ${{ matrix.arch }} -p ${{ env.NDK_LTS_VER }} -- build --verbose --features ffi
         env:
           ANDROID_NDK_HOME: ${{ github.workspace }}/android-ndk-r${{ env.NDK_LTS_VER  }}

--- a/README.md
+++ b/README.md
@@ -239,6 +239,9 @@ When running ``cargo build``, a static library called ``libquiche.a`` will be
 built automatically alongside the Rust one. This is fully stand-alone and can
 be linked directly into C/C++ applications.
 
+Note that in order to enable the FFI API, the ``ffi`` feature must be enabled (it
+is disabled by default), by passing ``--features ffi`` to ``cargo``.
+
 [thin C API]: https://github.com/cloudflare/quiche/blob/master/include/quiche.h
 
 Building
@@ -319,7 +322,7 @@ You can build the quiche library using the following procedure. Note that
 `--target` and `--android-platform` are mandatory.
 
 ```bash
- $ cargo ndk --target aarch64-linux-android --android-platform 21 -- build
+ $ cargo ndk --target aarch64-linux-android --android-platform 21 -- build --features ffi
 ```
 
 See [build_android_ndk19.sh] for more information.
@@ -383,13 +386,13 @@ To build quiche for iOS, you need the following:
 To build libquiche, run the following command:
 
 ```bash
- $ cargo lipo
+ $ cargo lipo --features ffi
 ```
 
 or
 
 ```bash
- $ cargo lipo --release
+ $ cargo lipo --features ffi --release
 ```
 
 iOS build is tested in Xcode 10.1 and Xcode 11.2.

--- a/tools/android/build_android.sh
+++ b/tools/android/build_android.sh
@@ -22,5 +22,5 @@ PATH="${TOOLCHAIN_DIR}/arch/arm64/bin":"${TOOLCHAIN_DIR}/arch/arm/bin":"${TOOLCH
 for target in aarch64-linux-android arm-linux-androideabi armv7-linux-androideabi i686-linux-android
 do
     echo "> target $target..."
-    cargo build --target $target $*
+    cargo build --target $target --features ffi $*
 done

--- a/tools/android/build_android_ndk19.sh
+++ b/tools/android/build_android_ndk19.sh
@@ -27,5 +27,5 @@ for target in \
     i686-linux-android
 do
     echo "> buliding $target..."
-    cargo ndk --target $target --android-platform $API_LEVEL -- build $*
+    cargo ndk --target $target --android-platform $API_LEVEL -- build --features ffi $*
 done

